packages/shared/src/modules/contributors/libs/types/contributor-get-all-request-dto.type.ts

```diff
@@ -0,0 +1,7 @@
+import { type PaginationQueryParameters } from "../../../../index.js";
+
+type ContributorGetAllRequestDto = {
```

Comment:
it's not a request dto
```suggestion
type ContributorGetAllQueryParameters = {
```